src/test/ui/associated-types/defaults-unsound-62211-1.stderr

```diff
@@ -42,6 +42,12 @@ LL |     + Display = Self;
 error[E0277]: `T` doesn't implement `std::fmt::Display`
   --> $DIR/defaults-unsound-62211-1.rs:41:9
    |
+LL | trait UncheckedCopy: Sized {
+   |       -------------
+...
+LL |     + Display = Self;
+   |       ------- required by this bound in `UncheckedCopy`
```

Comment:
hmm, I see that `UncheckedCopy` is underlined because it is being cited here -- I have to say that when I first read these errors, though, I didn't make that connection, and I was confused

Reply:
These don't have a label because we used to display them even if they where in the same line. Now that we only show that span if they are on their own line we could probably add a label without it being too verbose.

Reply:
ah, that would be nice, yeah